nemo/collections/asr/modules/conformer_encoder.py

```diff
@@ -77,7 +85,7 @@ class ConformerEncoder(NeuralModule, Exportable):
             Defaults to 0.0.
     """
 
-    def input_example(self, max_batch=1, max_dim=256):
+    def input_example(self, max_batch=1, max_dim=1024):
```

Comment:
Is it necessary to have such large input dim? It will OOM and lead to slow export. Max_dim corresponds to time dim.

Reply:
made it 256.